:robot: Synchronize branch `release-1.15` to `upstream/release-1.15`

serverless-qe opened the pull request:
This automated PR is to make sure the forked project's `release-1.15` branch (forked upstream's `release-1.15` branch) passes a CI.

cardil reviewed (approved):
/lgtm

maschmid commented:
> go: gotest.tools/gotestsum@latest: gotest.tools/gotestsum@v1.12.3 requires go >= 1.23.0 (running go 1.22.9; GOTOOLCHAIN=local)
> [MAGE] 🔧 Go install "gotest.tools/gotestsum@latest" have failed: running "go install gotest.tools/gotestsum@latest" failed with exit code 1```

knative-prow-robot committed:
build: Update knative.dev/toolbox/magetasks dependency (#422)

Updates the Go module dependency on knative.dev/toolbox/magetasks to the latest version within the build directory.

Co-authored-by: Chris Suszyński <ksuszyns@redhat.com>

serverless-qe committed:
:vhs: Images generated
